templates/common/userDetailsReadOnly.tpl

```diff
@@ -0,0 +1,41 @@
+{**
+ * common/userDetailsReadOnly.tpl
```

Comment:
Should be `templates/common/userDetailsReadOnly.tpl` (include full path)

Reply:
fixed.